conf/scylla.yaml

```diff
@@ -855,3 +855,19 @@ rf_rack_valid_keyspaces: false
 # Maximum number of items in single BatchWriteItem command. Default is 100.
 # Note: DynamoDB has a hard-coded limit of 25.
 # alternator_max_items_in_batch_write: 100
+#
```

Comment:
nit: extra commented line was added

Also, please improve patch subject. needs to included module/class and preferably be readable as a sentance.
For example: `scylla.yaml: add recommended value for stream_io_throughput_mb_per_sec`